servicetalk-http-utils/src/main/java/io/servicetalk/http/utils/TimeoutHttpRequesterFilter.java

```diff
@@ -178,38 +177,7 @@ public StreamingHttpConnectionFilter create(final FilterableStreamingHttpConnect
             @Override
             public Single<StreamingHttpResponse> request(final StreamingHttpRequest request) {
                 final FilterableStreamingHttpConnection delegate = delegate();
-                return TimeoutHttpRequesterFilter.this.withTimeout(request,
-                        delegate.connectionContext().protocol().major() >= 2 ? delegate::request :
-                                r -> delegate.request(r).liftSync(new BeforeFinallyHttpOperator(
-                                        new TerminalSignalConsumer() {
-                                            @Override
-                                            public void onComplete() {
-                                                // noop
-                                            }
-
-                                            @Override
-                                            public void onError(final Throwable throwable) {
-                                                // noop
-                                            }
-
-                                            @Override
-                                            public void cancel() {
-                                                // In alignment with cancellation processing in
-                                                // LoadBalancedStreamingHttpClient, we need to close the HTTP/1.x
-                                                // connection when this filter is applied at connection level.
-                                                // Otherwise, a connection will be marked as "free" and can be selected
-                                                // for another request, racing with closure.
-                                                LOGGER.debug("{} closing this {} connection due to cancellation",
```

Comment:
love the code removal here ❤️ 

LOGGER is no longer used in this class -> remove?